netdeploy/remote/deployedNetwork.go

```diff
@@ -357,34 +357,33 @@ func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, g
 		return err
 	}
 
-	srcWallet := getGenesisAlloc(fileCfgs.SourceWalletName, genesis.Allocation)
-	if srcWallet.Address == "" {
-		return fmt.Errorf("error finding source wallet address")
-	}
+	var src basics.Address
+	var addr basics.Address
+	var poolAddr basics.Address
+	var sinkAddr basics.Address
 
-	rewardsPool := getGenesisAlloc("RewardsPool", genesis.Allocation)
-	if rewardsPool.Address == "" {
-		return fmt.Errorf("error finding source rewards ppol address")
-	}
+	srcWalletName := strings.ToLower(fileCfgs.SourceWalletName)
+
+	for _, alloc := range genesis.Allocation {
+		comment := strings.ToLower(alloc.Comment)
+		addr, err = basics.UnmarshalChecksumAddress(alloc.Address)
+		if err != nil {
+			return fmt.Errorf("failed to unmarshal %s address %w", alloc.Comment, err)
```

Comment:
nit: you might want to specify the actual address in the error.
i.e.
fmt.Errorf("failed to unmarshal '%s' address '%v' : %w", alloc.Comment, alloc.Address, err)